sdk/search/Azure.Search.Documents/tests/SearchIndexClientTests.cs

```diff
@@ -286,6 +286,22 @@ public async Task GetIndexesNextPageThrows()
             Assert.ThrowsAsync<NotSupportedException>(async () => await e.MoveNextAsync());
         }
 
+        [Test]
+        public async Task GetIndexNames()
+        {
+            await using SearchResources resources = await SearchResources.GetSharedHotelsIndexAsync(this);
+
+            SearchIndexClient client = resources.GetIndexClient();
+
+            bool found = false;
+            await foreach (string name in client.GetIndexNamesAsync())
```

Comment:
You could simplify this with a single `client.GetIndexNamesAsync().Any(name => string.Equals(resources.IndexName, name, StringComparison.InvariantCultureIgnoreCase))`.

Reply:
That would be ideal, but it returns an `IAsyncEnumerable`, which LINQ methods don't seem to support.